auth/api/iam/api.go

```diff
@@ -382,25 +386,37 @@ func (r Wrapper) GetRequestJWT(ctx context.Context, request GetRequestJWTRequest
 	ro := new(jarRequest)
 	err := r.authzRequestObjectStore().Get(request.Id, ro)
```

Comment:
The request ID should be burned, to avoid client requesting it multiple times? (and causing load due to signing)

Reply:
Yes, as long as you add a TODO with a link to that issue.